src/GDriveTranslations/Config/Reader.php

```diff
@@ -41,6 +41,11 @@ private function forgeFromRaw(\stdClass $raw)
             $config->accessType = $raw->accessType;
         }
 
+        $config->sheetName = 'production';
+        if (isset($raw->sheetName)) {
+            $config->sheetName = $raw->sheetName;
+        }
+
```

Comment:
maybe like this (oneliner, and no double assignment): 

$config->sheetName = $raw->sheetName ?: 'production';